src/cljs/ataru/virkailija/editor/components/toolbar.cljs

```diff
@@ -8,20 +8,24 @@
    [taoensso.timbre :refer-macros [spy debug]]))
 
 (def ^:private toolbar-elements
-  (cond->
-    [["Lomakeosio" component/form-section]
-     ["Pudotusvalikko" component/dropdown]
-     ["Painikkeet, yksi valittavissa" component/single-choice-button]
-     ["Lista, monta valittavissa" component/multiple-choice]
-     ["Tekstikenttä" component/text-field]
-     ["Tekstialue" component/text-area]
-     ["Vierekkäiset tekstikentät" component/adjacent-fieldset]]
-    (fc/feature-enabled? :attachment)     (conj ["Liitepyyntö" component/attachment])
-    (fc/feature-enabled? :question-group) (conj ["Kysymysryhmä" component/question-group])
-    true                                  (conj ["Infoteksti" component/info-element])
-    (fc/feature-enabled? :question-group) (conj ["Pohjakoulutusmoduuli" base-education-module/module])
-    (fc/feature-enabled? :question-group) (conj ["Pohjakoulutusmoduuli (kk-yhteishaku)" kk-base-education-module/module])
-    true                                  (conj ["Ilmoitus riittämättömästä pohjakoulutuksesta" component/pohjakoulutusristiriita])))
+  (remove nil?
+          [["Lomakeosio" component/form-section]
+           ["Pudotusvalikko" component/dropdown]
+           ["Painikkeet, yksi valittavissa" component/single-choice-button]
+           ["Lista, monta valittavissa" component/multiple-choice]
+           ["Tekstikenttä" component/text-field]
+           ["Tekstialue" component/text-area]
+           ["Vierekkäiset tekstikentät" component/adjacent-fieldset]
+           (when (fc/feature-enabled? :attachment) ["Liitepyyntö" component/attachment])
```

Comment:
Tarviikohan näitä featureswitchejä enää kuljettaa mukana, vois vaan olettaa että ovat päällä?

Reply:
Mun puolesta vois kyllä, mietin sitä itsekin.